mist-api/src/lib/RouteHelper.js

```diff
@@ -90,15 +90,15 @@ class RouteHelper {
 			}
 
 			this.model_.findOne(criteria)
-			.then((entity) => {
-				if (!entity) {
-					next(errors.notFoundError)
-					return
-				}
-
-				res.json(entity)
-			})
-			.catch(next)
```

Comment:
This is a style concern and given that all our current formatting follows the indent pattern above, please do not change this. If we elect as a team to change it at a later time, then we should make a single PR dedicated to fixing that across the project.

Reply:
this was pointed out by the eslinter config by our team... I just fixed because it was telling me to :)

Reply:
K. Will have to take a look at the linter rules.